users/vieting/experiments/switchboard/hybrid/feat/specaug_sorted.py

```diff
@@ -0,0 +1,153 @@
+
+def sort_filters_by_center_freq(x, return_sorted_filters=True):
```

Comment:
This function is only used to get the indices here. I suggest to remove the option `return_sorted_filters` and to always return the indices. Also, the function name should then reflect that it returns sorted indices.